js/windowMgr.js

```diff
@@ -102,9 +103,10 @@ function getParsedUrl(appUrl) {
  * @param initialUrl
  */
 function createMainWindow(initialUrl) {
-    getMultipleConfigField([ 'mainWinPos', 'isCustomTitleBar', 'locale' ])
+    getMultipleConfigField([ 'mainWinPos', 'isCustomTitleBar', 'locale', 'devToolsEnabled' ])
```

Comment:
Can you move the argument array to a separate variable and pass that here? This is getting long.